content-security-policy/generic/generic-0_2.html

```diff
@@ -7,12 +7,19 @@
     <script src='/resources/testharnessreport.js'></script>
     <script nonce='abc'>
       var t_spv = async_test("Should fire violation events for every failed violation");
-      window.addEventListener("securitypolicyviolation", t_spv.step_func_done(function(e) {
-          assert_equals(e.violatedDirective, "script-src");
-      }));      
+      window.addEventListener(
+        "securitypolicyviolation", t_spv.unreached_func("securitypolicyviolation should not be emitted"));
```

Comment:
This expectation seems to have flipped, was this test incorrect to begin with?

Reply:
Yes, it was incorrect. There are no CSP violations, so the event should not have been emitted at all (regardless of the value of the `violatedDirective` property). Also, the sub-test was written to complete in two different ways: via a subsequent inline script and via the `step_func_done` method.

Reply:
Ok